src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -1178,22 +1182,21 @@ private static int GetDeclarationDepth(SyntaxNode? node)
                         return parentDepth;
                     }
 
-                    if (node.Parent is BlockSyntax ||
-                        (node is StatementSyntax && !(node is BlockSyntax)))
+                    if (node.Parent.IsKind(SyntaxKind.Block) ||
+                        (node is StatementSyntax && !node.IsKind(SyntaxKind.Block)))
                     {
                         // all nested statements are indented one level
                         return parentDepth + 1;
                     }
 
-                    if (node is MemberDeclarationSyntax ||
-                        node is AccessorDeclarationSyntax ||
-                        node is TypeParameterConstraintClauseSyntax ||
-                        node is SwitchSectionSyntax ||
-                        node is SwitchExpressionArmSyntax ||
-                        node is UsingDirectiveSyntax ||
-                        node is ExternAliasDirectiveSyntax ||
-                        node is QueryExpressionSyntax ||
-                        node is QueryContinuationSyntax)
+                    if (node.IsKind(SyntaxKind.TypeParameterConstraintClause) ||
+                        node.IsKind(SyntaxKind.SwitchSection) ||
+                        node.IsKind(SyntaxKind.SwitchExpressionArm) ||
+                        node.IsKind(SyntaxKind.UsingDirective) ||
+                        node.IsKind(SyntaxKind.ExternAliasDirective) ||
+                        node.IsKind(SyntaxKind.QueryExpression) ||
+                        node.IsKind(SyntaxKind.QueryContinuation) ||
```

Comment:
IsKind has overloads that take up to 8 args (I think)

Reply:
@alrz Yeah I wanted to use them. But they doesn't seem to be available in Compilers layer 😕

Reply:
@alrz I added these extensions in this layer. It made a more little performance improvement 🎉